content/SCALE/SCALETutorials/Network/ConfigureIPv6.md

```diff
@@ -51,6 +52,51 @@ Click on **Add** in the **Interfaces** to open the **Add Interface** screen.
 To access the UI after configuring an IPv6 address, enter the IPv6 address inside square brackets in the browser URL field.
 You cannot access the UI with the assigned host name when the system is configured on an IPv6 network.
 
+### Configuring Dual Stacking
+SCALE supports dual-stacking IPv4 and IPv6 addresses in the same interface.
+An IPv4 network cannot see or communicate with an IPv6 network unless some gateway is configured to allow IPv6 communication.
+Dual stacking these two protocols allows SCALE to see and communicate with an all IPv6 address or website.
+
+You must have IPv6 configured in your networking infrastructure.
+Add IPv6 to your network router to permit the incoming and outgoing traffic. This provides the required gateway to permit communication with this IP protocol.
+Assign a static IPv6 address and netmask, the network gateway address, and name servers to configure in SCALE.
+
+{{< hint type=info >}}
+When configuring dual stacking, the order in which you configure the two network IP protocols does not matter.
+{{< /hint >}}
+
+If IPv4 networking is already configured in SCALE, to set up dual stacking of IPv6 in the UI, go to **Network**:
+
+1. Add the IPv6 gateway information.
+
+   Click **Settings** on the **Global Configuration** widget. The **Edit Global Configuration** screen opens.
+
+   Enter the IPv6 address for the gateway in **IPv6 Default Gateway**.
+   
+   Click **Save**.
+
+2. Add the IPv6 static IP address to the primary interface.
+
+   Select the primary interface **en8s0**, then click **Edit**.
+
+   Click **Add*** to the right of **Aliases** to add another set of IP Address fields.
+
+   Enter the IPv6 address, then select the netmask.
+
+   Click **Save**
+
+3. Test the network change.
+   To verify the IPv6 address, in a new browser window, enter the address inside square brackets. For example, *[ffff:ff:59f8:100::12].
```

Comment:
is this supposed to be italicized?


```suggestion
   To verify the IPv6 address, in a new browser window, enter the address inside square brackets. For example, *[ffff:ff:59f8:100::12]*.
```

Reply:
yes in this instance this is an example or variable so italics is correct.